src/demo-app/input/input-demo.scss

```diff
@@ -0,0 +1,23 @@
+@import 'default-theme';
+@import 'variables';
+
+
+.material-icons {
```

Comment:
Prefix demo styles with `-demo`. This lets people inspect the elements in the demos and immediately nkow which styles are part of the component itself and which are just from the demo.